### DISCUSSION TITLE
Can Zwift Ride be used as a ‘dumb’ indoor bike without an app, using QZ?

### DISCUSSION THREAD

**Discussion by mwhrtin** [cagnulein/qdomyos-zwift#2919] [Q&A] [answered]
I’m considering replacing my current indoor spinning bike with the Zwift Ride. While Zwift is the main reason I’m exploring this switch, I’m hesitant to fully lock myself into the Zwift ecosystem. From what I’ve read, this is an issue that QZ helps to mitigate.

Another concern is that I want the ability to simply sit on the bike, start pedaling, set a desired resistance from the handlebar, and cycle for 5-10 minutes as a warmup without needing to launch a mobile app or any other software. However, from what I understand about the Zwift Ride, this isn’t possible because the Zwift Play controller requires the app to adjust the resistance on the trainer.

Would QZ be able to address this issue as well?

If I have QDomyos-Zwift installed on a Raspberry Pi that’s always running and ready to connect, would I then be able to adjust the resistance on the Zwift Ride without needing any other app to run?

**Comment by cagnulein** [comment, marked as answer]
> I’m considering replacing my current indoor spinning bike with the Zwift Ride. While Zwift is the main reason I’m exploring this switch, I’m hesitant to fully lock myself into the Zwift ecosystem. From what I’ve read, this is an issue that QZ helps to mitigate.

yes qz can give you full control of your bike




> Another concern is that I want the ability to simply sit on the bike, start pedaling, set a desired resistance from the handlebar, and cycle for 5-10 minutes as a warmup without needing to launch a mobile app or any other software. However, from what I understand about the Zwift Ride, this isn’t possible because the Zwift Play controller requires the app to adjust the resistance on the trainer.

exactly, you need qz for this



> If I have QDomyos-Zwift installed on a Raspberry Pi that’s always running and ready to connect, would I then be able to adjust the resistance on the Zwift Ride without needing any other app to run?

yes but raspberry for bluetooth it's not so reliable, and since with the zwift ride you will have at least 3 bluetotoh devices connected at the same time (1 trainer and 2 devices for the handlebar) i don't suggest to go in this direction. 

Instead you should simply use an old phone always on near to the bike. This is very reliable.

But in your scenario i will avoid to buy a zwift ride.
